index.html

```diff
@@ -90,15 +90,29 @@ <h2 id="introduction">1 Introduction</h2>
         <h3>1.1 What is Maestro?</h3>
 
         <p>
-          Maestro is a framework that enables rapid iteration for serverless
-          orchestration. The rise of serverless architectures and FaaS offerings
-          such as AWS Lambda has revolutionized how companies are developing
-          modern apps. The need for an orchestration layer over these
-          architectures has brought about services such as AWS Step Functions.
-          However, deploying apps that use Step Functions can be tedious and
-          error-prone. Maestro prioritizes speed and developer productivity by
-          automating this process so that the developer’s focus stays on
-          developing their application’s business logic.
+          Maestro is an open-source easy-to-use framework for quickly
+          bootstrapping, deploying, and rapidly iterating your serverless
+          orchestration (hence the name) workflows and applications using
+          Node.js® and AWS Step Functions.
+        </p>
+
+        <p>
+          Maestro makes it easy to get started developing your serverless
+          orchestration workflows, makes deploying a breeze, and provides a
+          method for quick re-deploys, helping you out not just in the initial
+          phase of your project, but in the ongoing maintenance and iteration as
+          well.
+        </p>
```

Comment:
Yes, I agree with the above. I specifically wanted to note that we need to carefully word the idea of re-deploying here since we mention re-deploying in the future work.

Reply:
OK, just pushed a change for this